src/opencmiss/zincwidgets/graphicseditorwidget.py

```diff
@@ -173,7 +176,7 @@ def _updateWidgets(self):
         self._pointScaleFactorsDisplay()
         self._ui.label_field_chooser.setField(labelField)
         # sampling attributes
-        if samplingattributes and samplingattributes.isValid():
+        if samplingattributes and samplingattributes.isValid() and self._graphics.getFieldDomainType() >=8:
```

Comment:
Probably better of having this condition as an `in` set of allowed domains rather than a random numeral.